### PR TITLE
Chapel 1.25 contributors file

### PR DESCRIPTION
**Pull request by bradcray**
Updates to the CONTRIBUTORS file reflecting people who have contributed to the Chapel 1.25 release.
Thanks to everyone who contributed, and also who helped double-check me!

### CODE REVIEW

**Comment by bradcray** [comment]
@VisionTheta , @strikeraryu, @shubhamkmr04, @king-11, @Yudhishthira1406 , @prashanth018 : Would you make sure I spelled and formatted your names correctly?

@king-11, @Yudhishthira1406 , @prashanth018: Would you also make sure I got your schools' names and hyperlinks correct?

@chapel-lang/core-contributors :
* If you're a new hire, please make sure I didn't mis-spell your name (or simply got it wrong... it's happened).
* If you merged changes from an external contributor for 1.25, please make sure they're credited here.

**Comment by lydia-duncan** [comment]
I think you're missing @Venkatavaradan-R

**Comment by bradcray** [comment]
Huh, that's odd... I primarily base my edits on https://github.com/chapel-lang/chapel/graphs/contributors?from=2020-12-15&to=2021-09-08&type=c and Venkat's name doesn't show up there for some reason.  For reference, we're talking about:
* https://github.com/chapel-lang/chapel/pull/17397
* https://github.com/chapel-lang/chapel/pull/17421
right?

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -4,36 +4,44 @@ Chapel Contributors
...
+* Shubham Kumar, individual contributor
 * Vassily Litvinov, [HPE]
 * David Longnecker, [HPE]
-* Aniket Mathur, [GSoC 2020] student from [Indian Institute of Technology Roorkee]
-* Divye Nayyar, individual contributor
+* Aniket Mathur, [GSoC 2021] mentor, [GSoC 2020] student from [Indian Institute of Technology Roorkee]
+* Ben McDonald, [HPE] (former intern from [Gonzaga University])
+* Divye Nayyar, [GSoC 2021] student from [Indian Institute of Technology Roorkee] (former individual contributor)
 * Sarah Nguyen, [HPE]
-* Yujia Qiao, [GSoC 2020] student from [Huazhong University of Science and Technology]
```
[the remark this answers is not in the document]
Comment (bradcray) [reply CONTRIBUTORS.md:29]: Thanks for catching this, Engin.  It looks like the March-today view I was using only covers commits made, not when those commits were merged, which is why I missed this one.  Apologies, @king-11 !

**Review by prashanth018 — CHANGES_REQUESTED** [review]
Could you change it to "Prasanth" instead of "Prashanth". Thanks.

**Comment by bradcray** [comment]
@prashanth018 : Sure thing.

**Comment by shubhamkmr04** [comment]
> @VisionTheta , @strikeraryu, @shubhamkmr04, @king-11, @Yudhishthira1406 , @prashanth018 : Would you make sure I spelled and formatted your names correctly?
> 
> @king-11, @Yudhishthira1406 , @prashanth018: Would you also make sure I got your schools' names and hyperlinks correct?
> 
> @chapel-lang/core-contributors :
> 
> * If you're a new hire, please make sure I didn't mis-spell your name (or simply got it wrong... it's happened).
> * If you merged changes from an external contributor for 1.25, please make sure they're credited here.

Yes, My name is correct in this. Thanks!

**Comment by VisionTheta** [comment]
> @VisionTheta , @strikeraryu, @shubhamkmr04, @king-11, @Yudhishthira1406 , @prashanth018 : Would you make sure I spelled and formatted your names correctly?
> 
> @king-11, @Yudhishthira1406 , @prashanth018: Would you also make sure I got your schools' names and hyperlinks correct?
> 
> @chapel-lang/core-contributors :
> 
> * If you're a new hire, please make sure I didn't mis-spell your name (or simply got it wrong... it's happened).
> * If you merged changes from an external contributor for 1.25, please make sure they're credited here.

Yes, my name is also correct, thank you.

**Comment by lydia-duncan** [comment]
> Huh, that's odd... I primarily base my edits on https://github.com/chapel-lang/chapel/graphs/contributors?from=2020-12-15&to=2021-09-08&type=c and Venkat's name doesn't show up there for some reason. For reference, we're talking about:
> 
>     * [Fixed minor formatting bug in primers/classes #17397](https://github.com/chapel-lang/chapel/pull/17397)
> 
>     * [Removed syntax highlighting for list of keywords in lexical-structure.rst #17421](https://github.com/chapel-lang/chapel/pull/17421)
>       right?

Yup, that's right!  I think that view tends to be sensitive to things like the email address used when making the commit - when someone on the team leaves and changes the email address associated with their account, we tend to not see commits they've made in that view with that old email address any more

**Comment by mppf** [comment]
Looks like we need to add Mohaned Mashaly @12mohaned for PR #17470.

**Comment by bradcray** [comment]
@mppf, @lydia-duncan, @e-kayrakli : I think I've fixed the oversights you noted.

@Venkatavaradan-R , @12mohaned , @prashanth018 : If you could check that I've got your name right / as you like it, that would be great.  Thanks!

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -4,36 +4,44 @@ Chapel Contributors
...
+* Nikhil Padmanabhan, [Yale University]
+* Gurudatta Patil, individual contributor
+* Ahmad Rezaii, [HPE]
 * Elliot Ronaghan, [HPE]
-* Mohammed Sharfuddin, individual contributor
+* Lakshya Singh, [GSoC 2021] student from [Indian Institute of Technology (BHU)]
```
[the remark this answers is not in the document]
Comment (bradcray) [reply CONTRIBUTORS.md]: No problem, Lakshya, and thanks for correcting me.  I'd seen it on their webpage, but went with what I was seeing on your pages.  Check that I got it right, if you would.

[the remark this answers is not in the document]
Comment (king-11) [reply CONTRIBUTORS.md]: Yes now its the correct one thanks

**Comment by bradcray** [comment]
@vrockz747 : Would you make sure I've got your name right?

**Pull request merged**